src/menu.scss

```diff
@@ -5,165 +5,303 @@
 $block: #{$fd-namespace}-menu;
 
 .#{$block} {
-
   // LOCAL VARS (set all themeable properties, always include !default)
-  // group title
-  $fd-menu-title-color: fd-color("text", 3) !default;
-  $fd-menu-title-background-color: fd-color("neutral", 1) !default;
-  $fd-menu-title-padding: fd-space("tiny") !default;
-
   // items
-  $fd-menu-item-color: fd-color("text", 2) !default;
-  $fd-menu-item-color--hover: $fd-color !default;
-  $fd-menu-item-background-color: map-get($fd-colors-background, 2) !default;
-  $fd-menu-item-background-color--hover: fd-color-state("hover") !default;
-  $fd-menu-item-background-color--selected: fd-color-state("selected") !default;
+  $fd-menu-item-color: var(--sapList_TextColor) !default;
+  $fd-menu-item-color--active: var(--sapList_Active_TextColor) !default;
 
-  $fd-menu-item-padding-x: fd-space("regular") !default;
-  $fd-menu-item-padding-y: fd-space("tiny") + 2px !default;
+  $fd-menu-item-background-color: var(--sapList_Background) !default;
+  $fd-menu-item-background-color--hover: var(--sapList_Hover_Background) !default;
+  $fd-menu-item-background-color--active: var(--sapList_Active_Background) !default;
+  $fd-menu-item-background-color--selected: var(--sapList_SelectionBackgroundColor) !default;
+  $fd-menu-item-background-color--selected-hover: var(--sapList_Hover_SelectionBackground) !default;
 
-  $fd-menu-item-separator-color: fd-color("neutral", 2) !default;
+  $fd-menu-item-separator-color: var(--sapGroup_ContentBorderColor) !default;
 
-  --fd-menu-item-separator-color: var(--fd-color-neutral-2);
-  --fd-menu-item-color: var(--fd-color-text-2);
-  --fd-menu-item-background-color: var(--fd-color-background-2);
-  --fd-menu-title-color: var(--fd-color-text-3);
-  --fd-menu-title-background-color: var(--fd-color-neutral-1);
+  $fd-menu-primary-icon-color: var(--sapContent_NonInteractiveIconColor) !default;
+  $fd-menu-primary-icon-color-active-selected: var(--sapList_Active_TextColor) !default;
 
   // BLOCK BASE *******************************************
   // set all BLOCK reset and baseline styles
   @include fd-reset();
-  @include fd-ellipsis();
 
-  position: relative;
-  padding-top: fd-space(base);
-  padding-bottom: fd-space(base);
   width: 100%;
-  display: inline-block;
 
-  &__group {
+  &__list,
+  &__sublist {
     @include fd-reset();
 
+    margin: 0;
+    padding: 0;
     list-style: none;
-    margin-bottom: 0;
-    padding-left: 0;
+    max-width: 20rem;
```

Comment:
Let's put `max-width` to parent `fd-menu`  container

Reply:
have moved to fd-menu